packages/plugins/rpc-manager/src/lib/eth_accounts.ts

```diff
@@ -26,7 +26,8 @@ export default class EthAccounts extends RpcModifier {
   }
 
   private async init() {
-    await this.nodeAccounts;
+    const nodeAccounts = await this.nodeAccounts;
+    this.rpcModifierEvents.request2("nodeAccounts:updated", nodeAccounts);
```

Comment:
is it missing an await

Reply:
Not really needed. This is fire and forget (the call after that is just a register and there is no need to wait before calling it)